documentation/docs/release-notes/2.44.1.md

```diff
@@ -0,0 +1,36 @@
+# Percona Monitoring and Management 2.44.1
+
+| **Release date** | April 24, 2025 |
+|------------------|---------------|
+| **Installation** | - [Install Percona Monitoring and Management 2](../quickstart/index.md)  <br> - [Install Percona Monitoring and Management 3](https://per.co.na/pmm/quickstart) |
+
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+It enables you to observe the health of your database systems, explore new patterns in their behavior, troubleshoot them, and execute database management operations regardless of whether your databases are located on-premises or in the cloud.
+
+## Release summary
+
+PMM 2.44.1 is a maintenance release that improves MongoDB monitoring reliability and addresses security vulnerabilities through updated Docker images.
+
+## What's new
+
+### PMM v3 migration notice
+
+When you launch PMM 2, you’ll now see a notification prompting you to migrate to PMM v3, as PMM 2 is no longer actively developed.
+
+This highlights PMM v3’s enhanced security, improved stability, and overall smoother experience. It also includes a direct link to the migration documentation to help guide you through the process.
```

Comment:
This sentence read odd to me so I tried to reword a bit.  If I didn't capture the sentiment correctly feel free to reject!

```suggestion
Moving to PMM v3 provides significantly enhanced security, improved stability, and an overall smoother user experience. You will now be presented with a direct link to the migration documentation to help guide you through the process.
```